### ISSUE TITLE
Possible wrong evaluation for motion sensor return value 

### ISSUE THREAD

**Issue by nordblick2** [Samfox2/homebridge-domotiga#25]
Hey,
while testing I found following quite strange line: 
```javascript
var value = (Number(result) == 0) ? 1 : 0;
```
Is that correct? I would expect that detected motion will be represented by a 1 and not by an 0. 

Suggestion:
```javascript
var value = (Number(result) == 0) ? 0 : 1;
```

https://github.com/Samfox2/homebridge-domotiga/blob/4cbdff63e9a92c32a1ada9dd340fac79cba76593/index.js#L1549

**Comment by Samfox2** [comment]
Yes, looks like a typo. MotionDetected should be 1.

**Comment by Samfox2** [comment]
Fixed in version 0.0.28, thanks for your help.
